Assets/MixedRealityToolkit/StandardAssets/Shaders/MRTK_Wireframe.shader

```diff
@@ -27,7 +48,7 @@ Shader "Mixed Reality Toolkit/Wireframe"
 
             // We only target the HoloLens (and the Unity editor), so take advantage of shader model 5.
             #pragma target 5.0
-            #pragma only_renderers d3d11
+            #pragma only_renderers d3d11 //D3D 11 & 12
```

Comment:
I know this bug existed before your PR. But we should remove the "only_renderers d3d11" pragma and wrap "target 5.0" in this preprocessor:

#if defined(SHADER_API_D3D11)
            #pragma target 5.0
#endif

So we support other platforms (iOS, Android, etc.). Feel free to do this in your PR or I can open a bug.

Reply:
No, will do in this PR! I just was following the comment and assuming this shader was only officially supported for HL